dpxdt/tools/url_pair_diff.py

```diff
@@ -84,6 +84,12 @@ def run(self,
                 'height': 1024,
             }
         }
+
+        if FLAGS.width:
+            config_dict['viewportSize']['width'] = FLAGS.width
```

Comment:
Would prefer to call this `viewport_width` or `viewportSize_width` - something more precise.